runtime/ts/storage/in-memory-storage.ts

```diff
@@ -14,10 +14,10 @@ import {CrdtCollectionModel} from './crdt-collection-model.js';
 import {Id} from '../id.js';
 import {Type} from '../type.js';
 
+const storageCache: Map<string, InMemoryStorage> = new Map();
```

Comment:
Either keep the double underscore or add a TODO that indicates this can't continue to exist in present form - it'll need to be moved to some kind of runtime context object once we have an API.